src/index.js

```diff
@@ -1,22 +1,35 @@
-import React, { useRef, useEffect } from 'react';
-import PropTypes from 'prop-types';
-import jdenticon from 'jdenticon';
+import React, { useRef, useEffect } from "react";
+import PropTypes from "prop-types";
+import jdenticon from "jdenticon";
 
-const Jdenticon = ({ value = 'test', size = '100%' }) => {
+function Jdenticon({ size = "100%", value, style, className, config }) {
+  // ref for icon svg
   const icon = useRef(null);
+
+  useEffect(() => {
+    // configure jdenticon based on given config, if available
+    // (idea from https://github.com/slavik0329/react-jdenticon)
+    config && jdenticon.configure(config);
+  }, [config]);
+
   useEffect(() => {
+    // update icon based on value
     jdenticon.update(icon.current, value);
   }, [value]);
```

Comment:
from jdenticon docs:

`jdenticon.update(element|selector[, hash|value[, config]])`

config (from v2.2.0) specifies an optional [configuration object](https://jdenticon.com/js-api/P_jdenticon_config.html#properties).
If no configuration is specified, the [global configuration](https://jdenticon.com/js-api/P_jdenticon_config.html) is used.
For backward compatibility, this parameter can also be a padding value in the range of 0.0 to 0.5.


That seems to be a better option when having different icons with their own properties.